mobile_verifier/src/heartbeats.rs

```diff
@@ -285,5 +439,26 @@ async fn validate_heartbeat(
         return Ok((cell_type, proto::HeartbeatValidity::GatewayOwnerNotFound));
     }
 
+    let Some(coverage_object) = heartbeat.report.coverage_object() else {
```

Comment:
Should we be invalidating heartbeats based on coverage objects in Phase 1??

Reply:
Nope! Removed